test_stale_repos.py

```diff
@@ -532,13 +532,23 @@ def test_exempt_repos(self):
         Test that a repo is exempt if its name is in the exempt_repos list.
         """
         repo = MagicMock(name="repo", spec=["name", "html_url"])
-        repo.name = "exempt_repo"
-        exempt_repos = ["exempt_repo"]
         exempt_topics = []
 
-        result = is_repo_exempt(repo, exempt_repos, exempt_topics)
+        test_cases = [
+            ("exempt_repo", ["exempt_repo"], True),
+            ("data-repo", ["data-*", "conf-*"], True),
+            ("conf-repo", ["exempt_repo", "conf-*"], True),
+            ("conf", ["conf-*"], False),
+            ("repo", ["repo1", "repo-"], False),
```

Comment:
Add test case for no exemption list
```suggestion
            ("repo", ["repo1", "repo-"], False),
            ("repo", [""], False),
```